substrate/frame/broker/src/dispatchable_impls.rs

```diff
@@ -80,7 +80,7 @@ impl<T: Config> Pallet<T> {
 			last_committed_timeslice: commit_timeslice.saturating_sub(1),
 			last_timeslice: Self::current_timeslice(),
 		};
-		let now = frame_system::Pallet::<T>::block_number();
+		let now = <<<T as crate::Config>::Coretime as CoretimeInterface>::RealyChainBlockNumberProvider as BlockNumberProvider>::current_block_number();
```

Comment:
Please create a `BlockNumberProviderOf<T>` type alias for this, or a `now` function.